integrations/streamer_test.go

```diff
@@ -9027,6 +9028,59 @@ stream
 	}
 }
 
+func TestStream_LambdaNow(t *testing.T) {
```

Comment:
This test is great! I'd love to see what happens if we did something like

```
stream
    |from()
      .measurement('account')
    |eval(lambda: "expiration" < now())
      .as('now')
    |httpOut('TestStream_EvalNow')
```

I'd also be curious to know what happens when we try to write this to Influx

```
stream
    |from()
      .measurement('account')
    |eval(lambda: "expiration" < now())
      .as('now')
    |influxDBOut()
	.database('db')
	.retentionPolicy('rp')
	.measurement('m')
	.precision('s')
	.tag('key', 'value')
	.flushInterval(1ms)
```
See the `TestStream_InfluxDBOut` for an example of this. I suspect there might be issues when we turn the time value into line protocol.

Reply:
Thanks. I'll try out adding another test with this and I'll get back to you when I'm done.

Reply:
Hi! I don't understand what you want me to do with these tests actually. Because 
* ```|eval(lambda: "expiration" < now())``` is going to output a boolean, right? Not a time value
* Even if we get a time value, I don't understand the use for this, because in my opinion now() is intended to be used with other functions such as unixnano() or minute() or something else but never alone.

Anyway, I have committed the tests I just wrote and which fail, but in order to fix them I'll need to know what kind of output you would expect for them because it's not entirely clear for me :).

Thanks in advance

Reply:
Ah! meant to do this

```
...
    |eval(lambda: now())
      .as('now')
...
```


> Even if we get a time value, I don't understand the use for this, because in my opinion now() is intended to be used with other functions such as unixnano() or minute() or something else but never alone.

I agree that they'll be used with `unixNano`, but my concern is what happens if I try to use it as a field value. In particular, what happens when we try to set it as a field.

Reply:
I updated the tests. They are failing, but I don't know why. I have read some part of the code *tick/eval.go* but I don't understand a lot about it.
Here is the output of one of the failed tests:

```
--- FAIL: TestStream_EvalNowInfluxDBOut (0.00s)
	streamer_test.go:11032: digraph TestStream_EvalNowInfluxDBOut {
		stream0 -> from1;
		from1 -> eval2;
		eval2 -> influxdb_out3;
		}
	streamer_test.go:9171: got  exp db
	streamer_test.go:9174: got  exp rp
	streamer_test.go:9177: got  exp s
	streamer_test.go:9180: got 0 exp 1
```

Can you give me any pointers on this? I don't have any useful log to understand what's going on. I would have expected the test to at least create the database, but it seems not. It must mean I wrote them wrong.